api_app/db/repositories/resources_history.py

```diff
@@ -0,0 +1,46 @@
+from typing import List
+import uuid
+from azure.cosmos.aio import CosmosClient
+from db.repositories.base import BaseRepository
+from core import config
+from models.domain.resource import Resource, NewResourceHistoryItem
+from pydantic import parse_obj_as
+
+
+class ResourceHistoryRepository(BaseRepository):
+    @classmethod
+    async def create(cls, client: CosmosClient):
+        cls = ResourceHistoryRepository()
+        await super().create(client, config.STATE_STORE_RESOURCES_HISTORY_CONTAINER)
+        return cls
+
+    @staticmethod
+    def resource_history_query(resourceId: str):
+        return f'SELECT * FROM c WHERE c.resourceId = "{resourceId}"'
+
+    @staticmethod
+    def resource_history_with_resource_version_query(resourceId: str, resourceVersion: str):
+        return f'SELECT * FROM c WHERE c.resourceId = "{resourceId}" AND c.resourceVersion = "{resourceVersion}"'
```

Comment:
same comment

Reply:
Will add validation for special chars and guid